tests/test_delete.py

```diff
@@ -0,0 +1,68 @@
+from . import SCHEMA_PREFIX, token, client, connection, schemas_simple
+import datajoint as dj
+
+
+def test_delete_dependent_with_cascade(token, client, connection, schemas_simple):
+    schema_name = f'{SCHEMA_PREFIX}group1_simple'
+    table_name = 'TableB'
+    restriction = dict(a_id=0, b_id=11)
+    vm = dj.VirtualModule('group1_simple', schema_name)
+    print(getattr(vm, table_name) & restriction)
```

Comment:
Perhaps remove this print statement to be in line with all the other test cases for delete

Reply:
Ah yes, good catch. I've remove it now. Thanks.